pkg/tbtc/node.go

```diff
@@ -153,7 +156,7 @@ func (n *node) joinDKGIfEligible(seed *big.Int, startBlockNumber uint64) {
 				n.protocolLatch.Lock()
 				defer n.protocolLatch.Unlock()
 
-				result, _, err := n.dkgExecutor.Execute(
+				result, endBlock, err := n.dkgExecutor.Execute(
```

Comment:
s/endBlock/executionEndBlock

Reply:
Done in 54916c43296c6911d2229b5ad1af34509c3d77cf.